### PR TITLE
Capitalize Cockpit attribute in titles

### PR DESCRIPTION
**Pull request by adamlazik1**
{Cockpit} renders lowercased in Satellite build titles. Added new attribute to fix this issue and to be consistent with Camel Case titles throughout the guides.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)
* [x] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [x] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8, orcharhino 6.2 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/attributes-base.adoc [hunk 1]
```diff
@@ -33,7 +33,9 @@
 :awx: AWX
 :certs-generate: foreman-proxy-certs-generate
 :certs-proxy-context: foreman-proxy
+// Cockpit and CockpitTitle attributes differ in Satellite build
```
Comment (maximiliankolb) [inline guides/common/attributes-base.adoc:36]:
```suggestion
```

**Review by maximiliankolb — APPROVED** [review]
I'd personally rather move this comment to your git commit message as additional info on _why_ we've changed it. Other than that, LGTM!

**Comment by adamlazik1** [comment]
Moved comment to commit message and merging :+1:

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   2cab9d917..bbd11735a  3.1 -> 3.1
   c930d5c44..4adf08e01  3.2 -> 3.2
   4bcbd9546..c0083d482  3.3 -> 3.3
   ff9b07f76..45e1a5a90  3.4 -> 3.4
   f76a6675e..4ee15691d  3.5 -> 3.5

**Pull request merged**